### ISSUE TITLE
Mesh SSIDs

### ISSUE THREAD

**Issue by rubo77** [WIStudent/freifunk-ssids#15]
It would come handy to select each mesh-SSID for each community as well

Discussion: https://forum.freifunk.net/t/jede-community-eigene-ad-hoc-ssid/13663

**Comment by WIStudent** [comment]
I don't think that collecting the mesh-SSIDs in this repository would be a good idea. This repository was primarily created to make updating the SSID list in the [Freifunk Auto Connect App](https://github.com/WIStudent/FreifunkAutoConnectApp) easier. To check for new SSIDs the app downloads the latest ssid.json file form the freifunk_auto_connect_production branch. This is done whenever the app is started and more than 24 hours have passed since the last check. This means that users who use the app on a daily basis will download the ssid.json file quite a lot. Therefore the size of that file should be kept as small as possible. Seeing that the Münsterland community alone uses 65 different mesh-SSIDs I fear that adding all mesh-SSIDs of every community would cause the file size to explode.

But you could still fork this repository and add the mesh-SSIDs there if you want.

**Comment by WIStudent** [comment]
Another alternative would be to collect the mesh-SSIDs in this repository but in a separate file.

**Comment by rubo77** [comment]
yes, a separate file would be fine. so you don't have to download it in your app
